db/migrate/20210518152311_move_image_uploads_to_carrierwave_path.rb

```diff
@@ -0,0 +1,31 @@
+# move any previously uploaded images not already inside 'original' version directory for Carrierwave to new location
+class MoveImageUploadsToCarrierwavePath < Mongoid::Migration
+  def self.up
+    branding_images = UserAssetService.get_local_assets(asset_type: :branding_images)
+    files_by_dir = {}
+    branding_images.each do |pathname|
+      dir_path, filename = pathname.split
+      files_by_dir[dir_path] ||= []
+      files_by_dir[dir_path] << filename
+    end
```

Comment:
you probably know the above could have been 
`files_by_dir = branding_images.reduce({}) do |hash, pathname|  ...`  
but not worth worrying about here

Reply:
I didn't realize you could pass an object/container to `reduce` - I thought it only took an integer or symbol for a method.